packages/geo/src/lib/catalog/catalog-browser/catalog-browser-layer.component.ts

```diff
@@ -44,6 +49,10 @@ export class CatalogBrowserLayerComponent {
 
   constructor() {}
 
+  ngOnInit(): void {
+    this.tooltip = this.computeTooltip(this.isInResolutionsRange());
```

Comment:
At the moment, isInResolutionRange computes the tooltip then it's computed again. See my comment below about the tooltip computation.